tests/test_helper.bash

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+if [ -z "$STACKER_NAMESPACE" ]; then
```

Comment:
Good idea. I think in CI we'll have a set prefix, and then append the build ID on it so that concurrent builds can't effect one another.